### ISSUE TITLE
Docker build fails, ERROR: No matching distribution found for pandas==1.1.2

### ISSUE THREAD

**Issue by rafee** [oracle/oci-designer-toolkit#409]
I am running the new MacBook Air with M1 processor. With high likelihood this is a problem related to docker arch. But I assumed you still might want to know about it.

```
ERROR: Could not find a version that satisfies the requirement pandas==1.1.2 (from versions: 0.1, 0.2b0, 0.2b1, 0.2, 0.3.0b0, 0.3.0b2, 0.3.0, 0.4.0, 0.4.1, 0.4.2, 0.4.3, 0.5.0, 0.6.0, 0.6.1, 0.7.0, 0.7.1, 0.7.2, 0.7.3, 0.8.0rc1, 0.8.0, 0.8.1, 0.9.0, 0.9.1, 0.10.0, 0.10.1, 0.11.0, 0.12.0, 0.13.0, 0.13.1, 0.14.0, 0.14.1, 0.15.0, 0.15.1, 0.15.2, 0.16.0, 0.16.1, 0.16.2, 0.17.0, 0.17.1, 0.18.0, 0.18.1, 0.19.0, 0.19.1, 0.19.2, 0.20.0, 0.20.1, 0.20.2, 0.20.3, 0.21.0, 0.21.1, 0.22.0, 0.23.0, 0.23.1, 0.23.2, 0.23.3, 0.23.4, 0.24.0, 0.24.1, 0.24.2, 0.25.0, 0.25.1, 0.25.2, 0.25.3, 1.0.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4, 1.0.5, 1.1.0, 1.1.1, 1.1.2, 1.1.3, 1.1.4, 1.1.5)
#5 60.42 ERROR: No matching distribution found for pandas==1.1.2
```

**Comment by toxophilist** [comment]
@rafee I would have thought this was more likely networking / access to pip site. The Docker build should simply use the latest OL7 for it's OS. I've tried a rebuild on my mac but can't repo this issue.

**Comment by rafee** [comment]
@toxophilist , I thought so initially too. However, two things changed my mind. This works flawlessly in MacBook with x86 architecture, as well as Linux machines. While Docker does allow x86 images to be installed in M1 machines, it also says support is rudimentary. All in all, I am not entirely sure, whether this is a bug report for OKIT or Docker. But, I would say, from OKIT perspective, one solution will be to upgrade to later versions of Pandas, which is more compatible with M1.

**Comment by rafee** [comment]
> @rafee I would have thought this was more likely networking / access to pip site. The Docker build should simply use the latest OL7 for it's OS. I've tried a rebuild on my mac but can't repo this issue.

Also, is your Mac M1 version? I was able to replicate this on M1 machines only.

**Comment by toxophilist** [comment]
@rafee I'm actually looking at dropping pandas so this should also resolve the issue.

**Comment by rafee** [comment]
Yes, that's another way to solve this. But while you are at it, you may need to take a look whether there's some other dependency that's causing problem with M1. I would be open to run compatibility for you, if you need access to an M1 machine

**Comment by jimmycal** [comment]
Any update on this issue, I'm seeing the same issue on my M1 Mac
